mappings/net/minecraft/MultiplayerInteractionManager.mapping

```diff
@@ -1,2 +1,15 @@
 CLASS net/minecraft/class_608 net/minecraft/MultiplayerInteractionManager
+	FIELD field_2608 miningPosX I
```

Comment:
maybe `breakingPosX` so we have a consistent terminology and dont switch between mining and breaking (like yarn (: ), technically `currentBreakingPosX` would be the closest in that case, but that feels obscenely long. Block breaking feels more appropriate then mining imo